orca-queue-redis/src/test/kotlin/com/netflix/spinnaker/orca/q/redis/RedisQueueSpec.kt

```diff
@@ -50,3 +59,29 @@ private val createQueue = { clock: Clock,
 private fun shutdownCallback() {
   redis?.destroy()
 }
+
+class ConvertToMessageSpec : Spek({
+  describe("should support deserializing a nested message") {
```

Comment:
Add it to `QueueSpec` and it will run as part of `RedisQueueSpec`. Am I missing something?